src/libs/print_settings.c

```diff
@@ -72,7 +72,7 @@ typedef struct dt_lib_print_settings_t
   int unit;
   int v_intent, v_pintent;
   char *v_iccprofile, *v_piccprofile, *v_style;
-  gboolean v_style_append;
+  gboolean v_style_append, v_black_point_compensation;
```

Comment:
This is going to invalidate old presets. I don't think that it is a problem right now since it's a new and unreleased module (and not an iop), but soonish you should think about doing that with `legacy_params`.